erikiado/jp2_online
# Task/create estudio socioeconomico

#### What's this PR do?

It connects the button on the interface to the creation of a study.

#### Where should the reviewer start?
In captura/views.py

#### How should this be manually tested?
Log in as a capturista, click the crear nuevo estudio button, and check that it appears on the screen,

#### Any background context you want to provide?
When this pull request gets accepted, the rest of the 


This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
